app/services/users.js

```diff
@@ -0,0 +1,17 @@
+const User = require('../models/').User,
+  errors = require('../errors');
+
+exports.emailIsAvailable = email => {
+  return User.findOne({
+    where: {
+      email
+    }
+  })
+    .then(user => {
+      if (user) return Promise.resolve(false);
+      return Promise.resolve(true);
```

Comment:
there is no need to use Promise.resolve here, just return it. Also and there is no need to return true, or false. You can just evaluate wether the response from this function is null or not. Also you either go for the  "put everything in model methods" approach or "create a service for each model" approach, dont mix